src/aap_eda/core/models/mixins.py

```diff
@@ -0,0 +1,21 @@
+from django.utils import timezone
+
+
+class ModifiedAtUpdaterMixin:
+    """Mixin to ensure that the `modified_at` field is updated.
+
+    Ensure that the `modified_at` field is updated when the model
+    is saved with specific fields (`update_fields`) specified for update.
+    Only works for `auto_now` fields.
+    """
+
+    def save(self, *args, **kwargs):
+        for field in self._meta.fields:
+            if (
+                field.name == "modified_at"
+                and hasattr(field, "auto_now")
+                and field.auto_now
+                and kwargs.get("update_fields", [])
+            ):
+                self.modified_at = timezone.now()
```

Comment:
Given the "fail safe" nature of the implementation it's the kind of change that  could be incorporated as a default for everything; though I can imagine that some code might not want that behavior as a default.  In which case there'd have to be a conscious decision to include it and that could be used to support the argument that the checking isn't necessary.

Reply:
I preferred to be conservative. I think for modified_at field we can do it always, but there are some "updated_at" that has not the auto_now set (which is something that I want to change later because it is wrong) 

But I would agree with you, as long as we are adding the mixin on purpose, the check may be not necessary.